imports/voting/ui/components/ProposalFilter.vue

```diff
@@ -2,14 +2,20 @@
   <div class="proposals-filter">
     <div class="container">
       <nav>
-        <button
-          v-for="f in filters"
-          :key="f"
-          :class="{active: currentFilter === f}"
-          @click="setFilter(f)"
+        <router-link
+          to="/"
+          tag="button"
+          :class="$route.path === '/' && 'active'"
         >
-          {{f}}
-        </button>
+          Newest
+        </router-link>
+        <router-link
+          to="/sorting/popular"
+          tag="button"
+          :class="$route.path === '/sorting/popular' && 'active'"
+        >
+          Popular
+        </router-link>
```

Comment:
This approach leads to a lot of duplicate code, I think better to generate this by an array of available filters.